.gitignore

```diff
@@ -38,7 +38,7 @@ test/dist/
 
 # this file is generated and shouldn't be checked in
 src/decrypter-worker.worker.js
+src/mse/transmuxer-worker.worker.js
```

Comment:
so that when we switch between 1.x and master branch, the worker file is always ignored.